Automattic/wp-calypso
# Anchor cleanup: remove from stepper

## Proposed Changes
Removes all Anchor related code from stepper.

## Testing Instructions
* Look at the code
* Verify tests run properly
* Go through some stepper flows to verify it's working correctly

Closes https://github.com/Automattic/wp-calypso/issues/75387
